definitions/divorce/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -1485,7 +1485,7 @@
     "TabID": "aosDetails",
     "TabLabel": "AOS",
     "TabDisplayOrder": 3,
-    "CaseFieldID": "AosLetterHolderId",
```

Comment:
are you sure this should go to the core file?

It will immidiatelly change prod and I think your story should be released in RRJ release.